lms/djangoapps/lti_provider/migrations/0004_add_version_to_graded_assignment.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name, missing-docstring, unused-argument, unused-import, line-too-long
```

Comment:
FYI automated testing will ignore linting on migrations files; this is a noted limitation with running diff-quality locally (if you do `paver run_pylint; paver run_quality` you will actually get the accurate report that we use). You shouldn't have to edit migrations files manually.